### PR TITLE
Pass golint check for pkg/descheduler/strategies/

### PR DESCRIPTION
**Pull request by lixiang233**

### CODE REVIEW

File: pkg/descheduler/strategies/lownodeutilization.go [hunk 1]
```diff
@@ -33,19 +33,25 @@ import (
...
 
 const (
+	// MinResourcePercentage is the minimum value of a resource's percentage
 	MinResourcePercentage = 0
+	// MaxResourcePercentage is the maximum value of a resource's percentage
 	MaxResourcePercentage = 100
 )
 
+// LowNodeUtilization evicts pods from overutilized nodes to underutilized nodes. Note that CPU/Memory requests are used
+// to calculate nodes' utilization and not the actual resource usage.
```
Comment (lixiang233) [inline pkg/descheduler/strategies/lownodeutilization.go:54]: Is this ok? @seanmalloy

**Comment by ingvagabund** [comment]
@lixiang233 excelent!!! I like how you turned the exported functions into private.

**Pull request merged**